ui/core/components/inputs/buffs_debuffs.ts

```diff
@@ -398,6 +398,16 @@ export const RetributionAura = withLabel(
 	}),
 	'Retribution Aura',
 );
+export const SanctityAura= withLabel(
+	makeBooleanRaidBuffInput({
+		actionId: (player) => player.getMatchingSpellActionId([
+			{ id: 20218, minLevel: 30 },
+		]),
+		fieldName: 'sanctityAura',
+		showWhen: (player) => ((player.getFaction() == Faction.Alliance) && ((player.getClass() == Class.ClassPaladin) || (player.getClass() == Class.ClassPriest))),
```

Comment:
leave only the faction check in here. The class checks should not be here but instead when adding this buff to the main buff/debuffs config it should be configured to be based on HolyPower stats (instead of empty stat array as you have it). This will propery add this buff to every sim that has HolyPower in its EPs

Reply:
Actually you dont even need the faction check as well as it looks like something that is also handled from the other configuration